new-pipeline/src/dashboards.js

```diff
@@ -1048,8 +1048,12 @@ function updateOSs() {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = "https://georgf.github.io/fx-data-explorer/index.html?search=" + metric + "&searchtype=in_name&optout=false&channel=" + channel + "&constraint=is_in&version=any"
-  var imgUrl = "https://upload.wikimedia.org/wikipedia/commons/6/64/Icon_External_Link.png"
   var descr = description === null ? metric : description
 
-  return " <a href=\"" + metricUrl + "\" target=\"_blank\">" + descr + "<img src=\"" + imgUrl + "\"/></a>"
+  var div = document.createElement('div');
+  div.classList.add("text-center");
+  div.classList.add("py-5");
```

Comment:
I don't think these two classes are needed, let's remove them.

Reply:
We do need `text-center` otherwise the descriptions come to the left.
I'll remove `py-5` for sure.